File: Java/Car.java
package Java;

public class Car {
    Integer id;
    String license; // variable global
    Account driver;
    Integer passenger;

    // Metodo constructor 

    public Car(String license, Account driver){ // variables locales 
        this.license = license; // Colocar la palabra reservada this para diferenciar de variable Local a global ya que manejan el mismo nombre.
        this.driver = driver;
    }


    //  Metodo para traer los datos

    void printDataCar() {
        System.out.println("License: " + license + "Name Driver: " + driver.name);
    }
}

File: Js/Car.js
//function Car(license, driver){  // colocamos los parametros que queremos poner. Tomo como si fuera un constructor y se lo pasa como parametro ya instancia directamente los datos.
class Car{
        constructor(license, driver){    
        this.id;
        this.license = license;
        this.driver = driver;
        this.password;
    }


    //Car.prototype.printDataCar = function(){
        printDataCar(){
        console.log(this.driver)
        console.log(this.driver.name)
        console.log(this.driver.document)
    }
}